src/ReferenceRepository/ObjectReferenceRepository.php

```diff
@@ -0,0 +1,75 @@
+<?php declare(strict_types=1);
+
+namespace Neusta\Pimcore\FixtureBundle\ReferenceRepository;
+
+/**
+ * based on \Doctrine\Common\DataFixtures\ReferenceRepository.
+ *
+ * @template T of object
```

Comment:
As already said in the other PR, we probably need to redo the templating here.

Reply:
Resolved in #14